iidm/iidm-xml-converter/src/main/java/com/powsybl/iidm/xml/util/IidmXmlUtil.java

```diff
@@ -7,17 +7,29 @@
 package com.powsybl.iidm.xml.util;
 
 import com.powsybl.commons.PowsyblException;
+import com.powsybl.commons.xml.XmlUtil;
+import com.powsybl.iidm.xml.AbstractNetworkXmlContext;
 import com.powsybl.iidm.xml.IidmXmlVersion;
 import com.powsybl.iidm.xml.NetworkXmlReaderContext;
+import com.powsybl.iidm.xml.NetworkXmlWriterContext;
+
+import javax.xml.stream.XMLStreamException;
+import java.util.Objects;
 
 /**
  * @author Miora Ralambotiana <miora.ralambotiana at rte-france.com>
  */
 public final class IidmXmlUtil {
 
+    public interface IidmXmlRunnable {
+        void run() throws XMLStreamException;
+    }
```

Comment:
Why don't you use the default Runnable? Maybe for exception handling? In that case, I think we have an UncheckedXmlStreamException